.github/workflows/check-vulnerability-whitesource.yml

```diff
@@ -44,36 +44,28 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ matrix.go-version }}
-      - uses: actions/checkout@v1
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_MAVEN_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_MAVEN_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
       - name: Vulnerability Scan
         id: vulnerability_scan
         env:
           wss_apikey: ${{ secrets.WSS_API_KEY }}
         run: |
           export PATH=$JAVA_HOME:$PATH
-          cd standalone-tools/vulnerability-scan
+          cd  tools/vulnerability-scan
           sudo yum install -y maven
           wget -q https://services.gradle.org/distributions/gradle-6.7-bin.zip
           sudo mkdir /opt/gradle
           sudo unzip -d /opt/gradle gradle-6.7-bin.zip
-          wget -qO - https://rpm.nodesource.com/setup_14.x | sudo bash -
-          sudo yum install -y nodejs
-          wget -qO - https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
-          sudo rpm --import https://dl.yarnpkg.com/rpm/pubkey.gpg
-          sudo yum install yarn -y
+          wget -qO - https://rpm.nodesource.com/setup_10.x | sudo bash -
+          sudo yum install -y nodejs-10.23.1
+          sudo npm install --global yarn@1.22.10
```

Comment:
Please do not hard code nodejs and yarn version.

Reply:
Dashboards plugins are throwing build errors, so had to match the node and yarn version mentioned in Dashboards.